binary/typedb

```diff
@@ -47,21 +47,34 @@ exit_if_java_not_found
 if [ -z "$1" ]; then
     echo "Missing argument. Possible commands are:"
     echo "  Server:          typedb server [--help]"
+    echo "  Node:            typedb node [--help]"
     echo "  Console:         typedb console [--help]"
     exit 1
 elif [ "$1" = "console" ]; then
     if [ -d "${TYPEDB_HOME}/console" ]; then
         SERVICE_LIB_CP="console/lib/*"
         CLASSPATH="${TYPEDB_HOME}/${SERVICE_LIB_CP}:${TYPEDB_HOME}/console/conf/"
     # exec replaces current shell process with java so no commands after this one will ever get executed
-        exec $JAVA_BIN ${CONSOLE_JAVAOPTS} -cp "${CLASSPATH}" -Dtypedb.dir="${TYPEDB_HOME}" com.vaticle.typedb.console.TypeDBConsole "${@:2}"
+        exec $JAVA_BIN ${JAVAOPTS} -cp "${CLASSPATH}" -Dtypedb.dir="${TYPEDB_HOME}" com.vaticle.typedb.console.TypeDBConsole "${@:2}"
     else
         echo "TypeDB Console is not included in this TypeDB distribution."
         echo "You may want to install TypeDB Console or TypeDB (all)."
         exit 1
     fi
-elif [[ "$1" = "server" ]] || [[ "$1" = "version" ]]; then
-    if [[ -d "${TYPEDB_HOME}/server" ]]; then
+elif [[ "$1" = "server" ]] || [[ "$1" = "node" ]]; then
+    if [[ "$1" = "server" ]]; then
+        TYPEDB_SERVER_NAME="TypeDB (Server)"
```

Comment:
Let's name this variable `TYPDB_NAME` and, for core let's just call it `"TypeDB"`.

Reply:
I think the value should be kept, since they are needed so that we can customise the error message which tells the user which distribution to install. In fact I've added another variable `TYPEDB_ALL_NAME` in order to make the error message even more precise.

Here's how we're using the variable:

```
...
if "server":
  TYPEDB_ALL_NAME="TypeDB (All)"
  TYPEDB_SERVER_NAME="TypeDB (Server)"
else if "node":
  TYPEDB_ALL_NAME="TypeDB Cluster (All)"
  TYPEDB_SERVER_NAME="TypeDB Cluster (Node)"
...
echo "$TYPEDB_SERVER_NAME is not included in this TypeDB distribution."
echo "You may want to install $TYPEDB_SERVER_NAME or $TYPEDB_ALL_NAME."
```